lib/parser.js

```diff
@@ -109,26 +113,22 @@ Parser.prototype._preprocess = function (node, context, config) {
 
       element.attribs.header = element.attribs.name || '';
       delete element.attribs.dynamic;
-      this.dynamicIncludeSrc.push({ from: context.cwf, to: element.attribs.src });
+      this.dynamicIncludeSrc.push({ from: context.cwf, to: actualFilePath, asIfTo: element.attribs.src });
```

Comment:
`to` refers to the actual file location (which might be in the boilerplate folder)

In a previous PR, we considered renaming it to `targetFilePath` and `actualFilePath` but that is inconsistent with the `staticIncludeSrc` etc and in cli's `collectIncludedFiles`, we use `dependency.to`